src/main/frontend/src/code-quarkus/pickers/extensions-picker.tsx

```diff
@@ -210,6 +210,7 @@ function Extension(props: ExtensionProps) {
 
 export const ExtensionsPicker = (props: ExtensionsPickerProps) => {
   const [filter, setFilter] = useState('');
```

Comment:
You could have this const outside of the ExtensionsPicker.

Reply:
Because of the change that I made to put in a prop, I changed the position to an outside the method in 'quarkus-project-utils.ts'.